### ISSUE TITLE
order of user actions on the lower left should be mobile, desktop, and accessibility and default to mobile

### ISSUE THREAD

**Issue by jasonfesta** [de-ai/playground.designengine.ai#123]
<img width="1552" alt="Screenshot 2019-12-16 at 3 32 42 PM" src="https://user-images.githubusercontent.com/1239243/70951977-b6b7ee00-2019-11ea-8ef0-23ad0b18906c.png">
